deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -821,6 +823,11 @@ <h4 class="panel-title pull-left pointer-cursor">
                         return false
                     }
 
+                    if ('root_volume_type' in clusterInfo.configs && (clusterInfo.configs['root_volume_type'] != "gp2" && clusterInfo.configs['root_volume_type'] != "gp3")) {
```

Comment:
lets move this to django config and use contains/in, similar to the internal implementation

Reply:
i know the rest of the section needs it to, there is also the encoding issue from django template to javascript, let me know what you think

Reply:
Is it fine if I create a separate work item for it to move such tasks to configs?

Reply:
https://jira.pinadmin.com/browse/CDP-5597 - work item for this